src/bindings/bnd_circle.cpp

```diff
@@ -24,12 +24,22 @@ void BND_Circle::SetCenter(ON_3dPoint center)
   m_circle.plane.UpdateEquation();
 }
 
-
 ON_3dPoint BND_Circle::PointAt(double t) const
 {
   return m_circle.PointAt(t);
 }
 
+BND_TUPLE BND_Circle::ClosestParameter(ON_3dPoint testPoint) const
+{
+  bool success = false;
+  double* t = 0;
+  success = m_circle.ClosestPointTo(testPoint, t);
+  BND_TUPLE rc = CreateTuple(2);
+  SetTuple(rc, 0, success);
+  SetTuple(rc, 1, t);
+  return rc;
+}
```

Comment:
This should be rewritten as
```
{
  double t = 0;
  bool success = m_circle.ClosestPointTo(testPoint, &t);
  ...
```

We want to declare a value as a double, not a pointer to a double. The address of this double is passed to m_circle.ClosestPointTo which lets that function modify the value of the double at that address in memory.